content/300-accelerate/650-troubleshoot.mdx

```diff
@@ -14,70 +14,69 @@ When working with Prisma Accelerate, you may encounter errors often highlighted
+In certain cases, a large number of records or fields are unintentionally fetched, which results in exceeding [the configured query response size limit](/accelerate/limitations#response-size-limit). This could happen when the [`where`](/orm/reference/prisma-client-reference#where) clause in the query is incorrect or entirely missing.
 
 **Suggested solution:** Double-check if the `where` clause is filtering the data as you expect. Consider using [pagination](/orm/prisma-client/queries/pagination) to prevent a large number of records from being fetched. Additionally, use the [`select`](/orm/reference/prisma-client-reference#select) clause to return only the fields that are being consumed to further reduce the response size.
 
 #### Fetching a large volume of data
 
 In many data processing workflows, especially those involving ETL (Extract-Transform-Load) processes or scheduled CRON jobs, there's a need to extract large amounts of data from data sources (like databases, APIs, or file systems) for analysis, reporting, or further processing. If you are running an ETL/CRON workload that fetches a huge chunk of data for analytical processing then you might run into this limit.
 
-**Suggested solution:** Consider splitting your query to fetch data in batches to prevent the 5MB limit from being reached. By splitting your query to retrieve data in batches, you ensure that each operation fetches only a portion of the total data volume, thereby not exceeding the size limit for a single fetch operation.
+**Suggested solution:** Consider splitting your query to fetch data in batches to prevent [the configured query response size limit](/accelerate/limitations#response-size-limit). By splitting your query to retrieve data in batches, you ensure that each operation fetches only a portion of the total data volume, thereby not exceeding the size limit for a single fetch operation.
 
 ## [`P6004`](/orm/reference/error-reference#p6004-querytimeout) (`QueryTimeout`)
 
-This error occurs when a database query fails to return a response within 10 seconds. The 10-second limit includes the duration of waiting for a connection from the pool, network latency to the database, and the execution time of the query itself. We enforce this limit to prevent unintentional long-running queries that can overload system resources.
+This error occurs when a database query fails to return a response within [the configured query timeout limit](/accelerate/limitations#query-timeout-limit). The query timeout limit includes the duration of waiting for a connection from the pool, network latency to the database, and the execution time of the query itself. We enforce this limit to prevent unintentional long-running queries that can overload system resources.
```

Comment:
Isn't the other obvious solution to change the response size limit?

Reply:
Nice catch 🙌 ! I added that as an solution underneath each problem.